src/__experimental__/components/grouped-character/grouped-character.component.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Textbox from '../textbox';
+import { generateGroups, toSum, toIndexSteps } from './grouped-character.utils';
+import Events from '../../../utils/helpers/events/events';
+
+const GroupedCharacter = (props) => {
+  const [pressedKey, updatePressedKey] = useState(null);
+
+  const { separator, groups, value } = props;
+  const stepIndices = groups.reduce(toIndexSteps, []);
+
+  const handleChange = (e) => {
+    const i = e.target;
```

Comment:
Couldn't this be? Plus `i` seems a bit vague for a name imo
```
 const i = ev.target;
 const { selectionEnd } = i;
```

Reply:
Agreed that `i` is vague, however it shouldn't matter which version I destructure the `selectionEnd` from.